src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-overview-dashboard/rgw-overview-dashboard.component.scss

```diff
@@ -0,0 +1,12 @@
+hr {
+  margin-bottom: 2px;
+  margin-top: 2px;
+}
+
+.list-group-item {
+  border: 0;
+}
+
+.pb-5 {
+  padding-bottom: 3.5rem !important;
+}
```

Comment:
is this necessary? this could break the layout in some views. You can test it out by resizing your window.

Reply:
Yes, this is necessary..tested with resizing the window..works fine